Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:18.17.1
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+ENV PORT=8080
```

Comment:
No. Don't do it, please.

Everything must be configured by the instance admin. What if we (and most probably) set up a CI/CD system for automated docker builds?

For env vars, we have `.env` file, that is read with txtdot. Or with docker-compose (by the way, it should be included in the repo).